book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :author, :title
+  attr_reader :rentals
+
+  def initialize(author, title)
+    @author = author
+    @title = title
+    @rentals = []
+  end
```

Comment:
## A book has many rentals: 
- Please a method for adding rentals should be present 
- Please open[ here to see the example](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)